docs/cli/introduction.md

```diff
@@ -9,40 +9,280 @@ next:
 
 # Introduction
 
-Terramate supercharges Terraform with Stacks, Orchestration, Git Integration, Code Generation, Data Sharing and more.
+This page provides a high-level overview of what Terramate CLI is and how it works.
 
-It focuses on improving the Developer Experience (DX), providing workflows, and lowering the time spent writing and maintaining infrastructure code for projects at any scale.
+If you want to get started with a *practical introduction* and learn about the Terramate CLI API, head over to the
+[Getting Started](./getting-started/index.md) guide.
 
-Compared to other prominent tooling in the market, **Terramate always generates native Terraform code**
-that integrates with existing tooling such as Terraform Cloud, Env0, Spacelift, Terragrunt, and others.
+<!-- To learn more about the motivation for Terramate, check out the Why Terramate? page. -->
```

Comment:
This is not rendered.

Reply:
that's on purpose :) I haven't built this page yet